src/pages/applications/Applications.jsx

```diff
@@ -1,9 +1,16 @@
 import React, { useEffect, useState } from "react";
 
-import { useParams,useNavigate } from "react-router-dom";
+import { useParams, useNavigate } from "react-router-dom";
 
 import axios from "axios";
-import { Pagination, Popover, Tag, Skeleton, Tooltip, notification } from "antd";
+import {
+  Pagination,
+  Popover,
+  Tag,
```

Comment:
why indentation & spaces are shown in review?
I've already said dont format the things which you've not worked! 
i dont  need spaces or indentation or formatter changes in review again @Ganeshdckap  @Parasuramandckap

Reply:
Please remove all the spaces & indentation in this PR which has been added by mistake using formatter and give it to me only the changes you've done @Ganeshdckap